pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html

```diff
@@ -0,0 +1,176 @@
+---
+$title: Release Schedule (screen-reader version)
+---
+{% do doc.styles.addCssFile('/css/components/molecules/tip.css') %}
+{% do doc.styles.addCssFile('/css/components/organisms/release-schedule-accessible.css') %}
+{% do doc.amp_dependencies.add('amp-bind', '0.1') %}
+
+<p>For a detailed description of the various release channels, see <a href="https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md">release-schedule.md</a>. There is also a <a href="./release-schedule.html">visual</a> (not screen-reader accessible) version of this page.</p>
```

Comment:
Would be good to add a bit more description here. 

Please link to the release schedule doc on amp.dev: `/documentation/guides-and-tutorials/learn/spec/release-schedule/?format=websites`. The links are missing margin as well:

<img width="225" alt="Screen Shot 2020-01-28 at 20 45 39" src="https://user-images.githubusercontent.com/380472/73300583-8adf7900-4211-11ea-9d91-8474725bb3cd.png">

Reply:
Done, hope this is a good enough description :)